mohtashami2021reproducibility/openreview/packages.tex

```diff
@@ -0,0 +1,13 @@
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage[T1]{fontenc}    % use 8-bit T1 fonts
```

Comment:
Please remove the fontenc package, as discussed in README. Bold/italics will work with ReScience fonts when this package is removed from the import

Reply:
Thank you very much. Done.